README.md

```diff
@@ -2,9 +2,6 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
-
-In the project directory, you can run:
 
 ### `npm start`
```

Comment:
- [ ] Kindly, Add a descriptive README file to your project,. A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally including `npm install`, `npm start` etc as this will be of help to non-technical users please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

-The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.